2.7.0/todo/result_distributed.md

```diff
@@ -30,9 +30,9 @@ The categories below are as follows:
 ### deprecation
 ### new features
 - Context parallel
-  - We provided a [Context Parallel API](https://github.com/pytorch/pytorch/pull/131351) for users to parallelize `torch.nn.functional.scaled_dot_product_attention` over the sequence dimension. We implemented
-  [Ring Attention](https://github.com/pytorch/pytorch/pull/131351) and an [AllGather-based approach](https://github.com/pytorch/pytorch/pull/132820) where the all-gather is issued before the first local SDPA
-  and the subsequent local SDPAs will have to wait until the all-gather completes, and offered a [user API](https://github.com/pytorch/pytorch/pull/142093) to select the desired approach. The implementation
+  - We provided a Context Parallel API ([#131351](https://github.com/pytorch/pytorch/pull/131351)) for users to parallelize `torch.nn.functional.scaled_dot_product_attention` over the sequence dimension. We implemented
```

Comment:
Just referencing PR by number should be sufficient, shouldn't it?
```suggestion
  - We provided a Context Parallel API (#131351) for users to parallelize `torch.nn.functional.scaled_dot_product_attention` over the sequence dimension. We implemented
```

Reply:
this is exactly my confusion. https://github.com/janeyx99/torch-release-notes/blob/main/2.7.0/todo/result_distributed.md says "use #PR_NUM to link to the PR, instead of [#PR_NUM](https://github.com/pytorch/pytorch/pull/#PR_NUM)" but I found the latter style is what actually be used in the file....